test/integration/query-interface.test.js

```diff
@@ -604,8 +604,9 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
     }
 
     describe('primary key', () => {
-      it('should add, read & remove primary key constraint', async function () {
-        await this.queryInterface.removeColumn('users', 'id');
+      (dialect !== 'yugabytedb' ? it : it.skip)('should add, read & remove primary key constraint', async function () {
+
+        await this.queryInterface.removeColumn('users', 'id'); // Removing a PRIMARY KEY column is not yet supported in Yugabyte
```

Comment:
let's split the 'remove' part in another test then, 'read' & 'add' should still be tested (unless they're also unsupported)

Reply:
So, can we add the new splitted test there itself which will run for yugabytedb only or should we write it separately for dialect specific?

Reply:
I think what would be best would be to split this test in 2:
- one for "is able to add & read primary key constraint"
- one for "is able to remove primary key constraint"

And only disable the second one if the dialect is `yugabytedb`

Reply:
Got that, but there is one issue like in the starting of the test we are removing the `PRIMARY KEY` column which cannot be possible in YugabyteDB so, need to modify the test for that as well. 
So, shall we need to modify the test in general or just add another modified test for `yugabytedb`?